src/common/components/Input/Icons/ViewOff.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const ViewOff = (
```

Comment:
I was thinking we could just put this in a ViewOff.svg file and just keep the `<svg>...</svg>` part.